string program_id
string name
string description
bool has_blockly
string saved_at
string python_code
string blockly_code